twilight-http/src/routing.rs

```diff
@@ -2842,15 +2714,18 @@ impl Display for Route<'_> {
             } => {
                 f.write_str("guilds/")?;
                 Display::fmt(guild_id, f)?;
-                f.write_str("/members/search?query=")?;
-                Display::fmt(&utf8_percent_encode(query, NON_ALPHANUMERIC), f)?;
+                f.write_str("/members/search")?;
 
-                if let Some(limit) = limit {
-                    f.write_str("&limit=")?;
-                    Display::fmt(limit, f)?;
-                }
+                let query = QueryString::new()
+                    .with_value(
+                        "query",
+                        // The builder is supposed to encode strings into UTF-8 format, however for some reason,
+                        // it doesn't handle slashes properly. Unclear if this is a bug in the library.
+                        utf8_percent_encode(query, NON_ALPHANUMERIC).to_string(),
+                    )
```

Comment:
`/` is actually allowed in query parameters without encoding.

That said the library is wrong in another way, because it uses the FRAGMENT identifiers instead of the QUERY identifiers, this means that `#` will be encoded and backtick (`) will not.

Reply:
Coming in from https://github.com/sunsided/query-string-builder/issues/11, what's the rationale about the _fragment_ identifier encoding here when encoding the query string (i.e., not the fragment)?